setup.py

```diff
@@ -274,7 +274,7 @@ def get_dynamic_setup_params():
         'install_scripts': InstallScriptsCommand,
         'sdist': SDistCommand,
     },
-    name='ansible',
+    name='ansible-base',
```

Comment:
FTR this specific part looks good to me. The confusion comes with `package_dir` and understanding the full picture and the mention of Git.